meta-lmp-base/classes/lmp-staging.bbclass

```diff
@@ -8,6 +8,8 @@
 INHERIT_KERNEL_MODSIGN = ""
 
 python __anonymous() {
+    pn = d.getVar('PN')
```

Comment:
I'd combine both commits in a single - introducing and using variables in the same commit is fine IMHO.

Reply:
This bitbake class has mainly workaround that will be reverted asap or when fixed upstream. In the first patch they went in the same commit, but have it in a separate patch add the possibility to revert one and not the other.
The `PN` is a very common variable so will always be there and meanwhile I already have new patches that makes uses  it.